### PR TITLE
#164798158 Update TravisCI to run tests

### PR DESCRIPTION
**Pull request by ezrogha**
#### What does this PR do?
Integrates TravisCI with a readme badge to the project

#### Description of Task to be completed?
Integrating TravisCI to author's Heaven to enable continuous integration

#### What are the relevant pivotal tracker stories?
[#164798158](https://www.pivotaltracker.com/story/show/164798158)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+node_js:
+  - "stable"
+
+cache:
+  directories:
+  - node_modules
+
+script:
+  - npm run test:coverage
```
Comment (dannylwe) [inline .travis.yml]: Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding "files.insertFinalNewline": true to your settings.json file

Comment (felixkiryowa) [inline .travis.yml]: @ezrogha  could put a space at the end of this file

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (ezrogha) [reply .travis.yml]: Thank you @dannylwe
I have worked on it

Comment (ezrogha) [reply .travis.yml]: Thank you @felixkiryowa 
I have worked on it

**Review by felixkiryowa — APPROVED** [review]
Good work

**Review by dannylwe — APPROVED** [review]
LGTM

**Pull request merged**